llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -164,6 +164,9 @@ class SPIRVInstructionSelector : public InstructionSelector {
                  unsigned comparisonOpcode, MachineInstr &I) const;
   bool selectCross(Register ResVReg, const SPIRVType *ResType,
                    MachineInstr &I) const;
+  bool selectClip(Register ResVReg, const SPIRVType *ResType,
```

Comment:
if we ened up renaming the intrinsic this should also rename to something like `selectDiscard`.